src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -70,8 +83,9 @@ public CheckResult Check(string expressionText, RecordType parameterType = null)
                 parameterType = new RecordType();
             }
 
-            var formula = new Formula(expressionText);
+            var formula = new Formula(expressionText, loc: Config.LanguageSettings);
```

Comment:
We don't want to enable locale-specific parsing yet, without writing a ton of tests. I think leaving out this for now is the right approach.

Reply:
Removed.